metadata-builders/src/test/java/com/linkedin/metadata/builders/search/DatasetIndexBuilderTest.java

```diff
@@ -52,9 +52,7 @@ public void datasetPropertiesSetsDescription() {
     final List<DatasetDocument> actualDocs = _indexBuilder.getDocumentsToUpdate(datasetSnapshot);
 
     // then
-    assertThat(actualDocs.size()).isEqualTo(2);
-    assertThat(actualDocs).contains(expectedDocument1);
-    assertThat(actualDocs).contains(expectedDocument2);
+    assertThat(actualDocs).containsExactly(expectedDocument1, expectedDocument2);
```

Comment:
looks like order matters for `containsExactly` but does it matter to us? Is `containsExactlyInAnyOrder` better?

Reply:
Good question. Let me think that over some more.

Reply:
I think since we're exposing a `List` rather than `Collection`, we should probably assert order.